azure/classes/az-lock.ps1

```diff
@@ -1,10 +0,0 @@
-Enum LockLevel {
-    CanNotDelete = 0
-    ReadOnly = 1
-    None = 2
-}
-Class AzLock{
```

Comment:
Removed for now until I can find a better way to handle as we start building deep with KV and LA